CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 ## Contributing
 
-We aspire to create a welcoming environment for collaboration on this project
+We apire to create a welcoming environment for collaboration on this project
```

Comment:
```suggestion
We aspire to create a welcoming environment for collaboration on this project
```

Reply:
Ah yes, I had changed this just to show how the CI would pick up the typo in https://github.com/JohnHillegass/sig-security/runs/1601873376?check_suite_focus=true